ingress/controllers/nginx/nginx.tmpl

```diff
@@ -73,6 +73,9 @@ http {
 
     client_max_body_size "{{ $cfg.bodySize }}";
 
+    # disable warning
+    uninitialized_variable_warn off;
```

Comment:
> isn't this dangerous?

No (uninitialized variables appear as empty in the log). I removed the section to avoid hiding this issue.